src/LibExpat.jl

```diff
@@ -213,12 +213,12 @@ function attrs_in_to_dict(attrs_in::Ptr{Ptr{UInt8}})
             i=i+1
             attr = unsafe_load(attrs_in, i)
 
-            if (attr == C_NULL) error("Attribute does not have a name!") end
+            attr == C_NULL && error("Attribute does not have a name!")
             v = unsafe_string(attr)
 
             attrs[k] = v
 
-            @DBG_PRINT("$k, $v in $name")
```

Comment:
not sure what name refers to since there is no local variable called name in this function